examples/Client/PublishSubscribe/BulkPublishEventExample/README.md

```diff
@@ -0,0 +1,34 @@
+# Dapr .NET SDK Bulk publish example
+
+## Prerequisites
+
+- [.NET Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download) installed
+- [Dapr CLI](https://docs.dapr.io/getting-started/install-dapr-cli/)
+- [Initialized Dapr environment](https://docs.dapr.io/getting-started/install-dapr-selfhost/)
+- [Dapr .NET SDK](https://docs.dapr.io/developing-applications/sdks/dotnet/)
+
+## Running the example
+
+To run the sample locally run this command in the DaprClient directory:
```

Comment:
This needs to be updated to show the controller you need, right?

Reply:
Yup, updated